views/authentication/forms/login.hbs

```diff
@@ -14,11 +14,13 @@
 				<input type="hidden" name="redirect" value={{redirect}} />
 
 				<div class="form-group">
-					<input type="text" autocomplete="off" name="username" class="form-control form-control-lg" placeholder="{{$t "login.input.emailUsername"}}" data-testid="username" required>
-					<div class="input-group mt-1" id="show_hide_password">
-						<input type="password" autocomplete="current-password" name="password" maxlength="72" class="form-control form-control-lg" placeholder="{{$t "login.input.password"}}" data-testid="password" required>
+					<label for="name">{{$t "login.input.emailUsername"}}</label>
+					<input type="text" id="name" autocomplete="off" name="username" class="form-control form-control-lg" placeholder="{{$t "login.input.emailUsername"}}" style="margin-bottom: 0.5rem" data-testid="username" required >
+						<label for="password">{{$t "login.input.password"}}</label>
+					<div class="input-group mt-1" id="show_hide_password" style="margin-top: 0rem !important">
+						<input type="password" id="password" autocomplete="current-password" name="password" maxlength="72" class="form-control form-control-lg"  placeholder="{{$t "login.input.password"}}" data-testid="password" required>
 						<div class="input-group-addon">
-							<a href=""><i class="fa fa-eye-slash" aria-hidden="true"></i></a>
+							<a title="{{$t "login.anchor.title.show_password"}}" style="cursor: pointer;"><i class="fa fa-eye-slash" aria-hidden="true"></i></a>
```

Comment:
I've noticed I had a problem interacting with a link if no href was provided. Did it work for you?

Reply:
you are right!